code/modules/clothing/under/solgov.dm

```diff
@@ -230,40 +212,33 @@
 	worn_state = "greenservice_com"
 
 //Dress
-/obj/item/clothing/under/mildress
```

Comment:
This should probably stay as mildress, because /obj/item/clothing/under/dress is defined elsewhere, and these almost certainly should not be inheriting from that item.